settings.js

```diff
@@ -119,11 +119,48 @@ function _addIcon(pages, itemId, index, itemsPerPage) {
     };
 }
 
+// AppDisplay._ensureDefaultFolders() adds two hard-coded folders before this
+// extension has a chance to override it. Remove them so they do not interfere
+// with our icon grid defaults mechanism.
+//
+// Refer to https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/48.0/js/ui/appDisplay.js?ref_type=tags#L2271-2287
```

Comment:
48.0 is a different version of gnome-shell than the extension currently targets – is this code the same in 43.9?

Reply:
Yes, they are the same for this part.  I try to find the 43.0 tag, but no luck.  However, we can get the same code from EOS' gnome-shell https://github.com/endlessm/gnome-shell/blob/13f27e27e593907f085745c3b79e77512b89fc0f/js/ui/appDisplay.js#L2286-L2302

Reply:
Ah!  Has to scroll down for 43.9

Reply:
Going to update the comment.

Reply:
Updated the comment in the new commit.